heltour/tournament/admin.py

```diff
@@ -823,11 +812,15 @@ def insert_alternates(alts_split):
                                              board_number=board_number)
 
         season = get_object_or_404(Season, pk=object_id)
+        season_started = bool(Round.objects.filter(season=season, publish_pairings=True).count())
+        team_count = Team.objects.filter(season=season).count()
         if request.method == 'POST':
-            form = forms.CreateTeamsForm(request.POST)
+            if season_started:
+                return HttpResponse(status=400)
```

Comment:
Shouldn't we just disable the entire view if season_started?

Reply:
You mean, move this to the top?